.github/workflows/test-credentials.yml

```
@@ -0,0 +1,43 @@
+name: Test Credentials
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+concurrency:
+  group: test-credentials-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test-gcp-credentials:
+    name: Test GCP credentials
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: google-github-actions/auth@v3
+        with:
+          credentials_json: ${{ secrets.TEST_GCP_CREDENTIALS }}
+      - uses: google-github-actions/setup-gcloud@v2
+      - run: gcloud config get-value project > /dev/null
+
+  test-gh-bot-token:
+    name: Test GitHub bot token
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - run: gh api user --jq '.login' --token "${{ secrets.TEST_GH_BOT_TOKEN }}"
```

I'm not sure if you've tried this successfully with a different version of the GH CLI, but for me the `--token` flag is not supported. I'm using `gh` version `2.45.0` and it gives the error `unknown flag: --token`.

According to the [GH CLI docs](https://cli.github.com/manual/gh_auth_login) we should set the `GH_TOKEN` environment variable to use `gh` in actions. Setting the env does works locally for me.

```suggestion
      - run: gh api user --jq '.login'
        env:
          GH_TOKEN: ${{ secrets.TEST_GH_BOT_TOKEN }}
```